systemverilog-plugin/UhdmAst.cc

```diff
@@ -3824,6 +3824,12 @@ void UhdmAst::process_tf_call(AST::AstNodeType type)
             current_node->children.push_back(node);
         }
     });
+    // prefer full name of function
+    // this fixes case when we imports single function from package
+    // but it calls another, not imported, function
+    if (vpiHandle function_h = vpi_handle(vpiFunction, obj_h)) {
+        current_node->str = get_name(function_h, true);
```

Comment:
Is it possible that we could have the same result by using `make_ast_node` with the argument `prefer_full_name` set to `true`?
```
AST::AstNode *UhdmAst::make_ast_node(AST::AstNodeType type, std::vector<AST::AstNode *> children, bool prefer_full_name)
```
It seems to that this way we would have the proper name set immediately, without the need to modify `node->str` after the node object is created.

Reply:
Unfortunately not, ``vpiFuncCall`` have only name without scope, but ``vpiFunction`` have full name:
```
       \_return_stmt: , line:7:5, endln:7:11
         |vpiParent:
         \_function: (pkg::func_1), line:6:3, endln:8:14
         |vpiCondition:
         \_func_call: (func_2), line:7:12, endln:7:20
           |vpiParent:
           \_return_stmt: , line:7:5, endln:7:11
           |vpiName:func_2
           |vpiFunction:
           \_function: (pkg::func_2), line:3:3, endln:5:14
 ```